packages/embark/src/cmd/cmd_controller.js

```diff
@@ -1,12 +1,14 @@
-import {__} from 'embark-i18n';
-import {dappPath, embarkPath} from 'embark-utils';
+import { Config, Engine, Events, fs, TemplateGenerator } from 'embark-core';
+import { __ } from 'embark-i18n';
+import { dappPath, embarkPath, joinPath, setUpEnv } from 'embark-utils';
 let async = require('async');
 const constants = require('embark-core/constants');
 const Logger = require('embark-logger');
 const {reset: embarkReset, paths: defaultResetPaths} = require('embark-reset');
-const fs = require('../lib/core/fs.js');
 const cloneDeep = require('clone-deep');
 
+setUpEnv(joinPath(__dirname, '../../'));
```

Comment:
`setUpEnv()` is the function that ensures `DAPP_PATH` and `NODE_PATH` are set correctly. We used to call it inside `env.js`. However, `env.js` used to be loaded in `lib/core/config` but since we moved that one to `embark-core` we need to ensure `setUpEnv()` is still called at a place when Embark is intialized (to ensure env variables are set correctly). That's why this is in `cmd_controller` now